.config/swift.config.php

```diff
@@ -0,0 +1,30 @@
+<?php
+if (getenv('OBJECTSTORE_SWIFT_URL')) {
+  $CONFIG = array(
+    'objectstore' => [
+      'class' => 'OC\\Files\\ObjectStore\\Swift',
+      'arguments' => [
+        'autocreate' => getenv('OBJECTSTORE_SWIFT_AUTOCREATE') ? true : false,
```

Comment:
The default in the server project is false[0]. Can we make this work like what we tested out in s3.config.php?

$autocreate = getenv('OBJECTSTORE_SWIFT_AUTOCREATE')

'use_path_style' => $autocreate == true && strtolower($autocreate) !== 'false'

[0]: https://github.com/nextcloud/server/blob/7e6f04490e286450d009c156f08de6a22f0f653d/lib/private/Files/ObjectStore/SwiftFactory.php#L62-L67

Reply:
Sure, will do